AirCasting/Dashboard/SessionListView.swift

```diff
@@ -14,14 +14,16 @@ struct SessionsListView: View {
     @FetchRequest<SensorThreshold>(sortDescriptors: [.init(key: "sensorName", ascending: true)]) private var thresholds
     @StateObject private var coreDataHook: CoreDataHook
     @Binding private var isRefreshing: Bool
+    @Binding private var isDownloading: Bool
     @InjectedObject private var featureFlagsViewModel: FeatureFlagsViewModel
     
     private let listCoordinateSpaceName = "listCoordinateSpace"
     private let selectedSection: DashboardSection
 
-    init(selectedSection: DashboardSection, isRefreshing: Binding<Bool>, context: NSManagedObjectContext) {
+    init(selectedSection: DashboardSection, isRefreshing: Binding<Bool>, isDownloading: Binding<Bool>, context: NSManagedObjectContext) {
```

Comment:
this can be a bit confusing. Without a good project understanding it will be hard to tell what's different between `isRefreshing` and `isDownloading`. Maybe `isDownloadingMeasurements`? (same for the property name)